service/service.go

```diff
@@ -72,6 +72,7 @@ func (c controllerConfigGetter) GetConfig() (types.DriverOptions, error) {
 			return driverOptions, err
 		}
 		driverOptions.StringOptions["rkeConfig"] = string(config)
+		driverOptions = rke.SetRotateCertsOptions(driverOptions, c.clusterSpec)
```

Comment:
why does it have to be done on the kontainer-engine side? All the information on how we want to rotate the certs, should be available in the rkeConfig, and therefore can be extracted on the rke side